bittorrent/torrent.go

```diff
@@ -799,37 +799,37 @@ func (t *Torrent) GetState() int {
 }
 
 // GetStateString ...
-func (t *Torrent) GetStateString() string {
+func (t *Torrent) GetStateString() (string, int) {
```

Comment:
Intead of double return values, I would just make it `GetState() (int)` and return status.
Then If needed, it can be explicitly resolved by `StatusStrings[GetState()]`

Reply:
i kind of missed that there is already `GetState() int`
https://github.com/elgatito/elementum/blob/52b05165e8532f6ed0a068ea33abaee431fc1327/bittorrent/torrent.go#L794-L799

it is not so sophisticated as `GetStateString()` but maybe it is ok for us?

Reply:
@elgatito 
i made changes, please take a look when you have time. thanks.